src/tests/simulation/src/ctl_ext.rs

```diff
@@ -299,11 +301,15 @@ impl Cluster {
                     "reschedule",
                     "--plan",
                     plan.as_ref(),
+                    "--revision",
+                    &format!("{}", revision),
                 ]);
                 risingwave_ctl::start(opts).await
             })
             .await??;
 
+        self.context.reschedule_revision += 1;
```

Comment:
Is it possible to retrieve the latest revision here instead of hardcoding `+1` here?

Reply:
fixed